templates/explore/organizations.tmpl

```diff
@@ -6,10 +6,16 @@
 
 		<div class="ui user list">
 			{{range .Users}}
+				{{if (or (eq .Visibility 1) (and ($.SignedUser) (or (eq .Visibility 2) (and (.IsUserOrgPartOf $.SignedUserID) (eq .Visibility 3)) ($.IsAdmin))))}}
```

Comment:
This will break paging as if most organizations are private can be that you will get empty pages or pages with very few orgs in it. These conditions should be already part of database query

Reply:
Good catch, when I did that I did not have that many user so did not see the pages. Everything was one page. Agree with your solution.

Reply:
@DblK I've just resolved the latest conflicts. I'm going on vacation until mid-august, do you want to take over this PR again, and add in the SQL conditions?